iCalRetriever.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.touch_actions import TouchActions
+from time import sleep
+
+driver = webdriver.Chrome()
+# driver.implicitly_wait(1)
+# Loading the hyper planning site
+driver.get("https://hplanning2016.umons.ac.be/invite")
```

Comment:
@DocSkellington J'ai un problème de version de chrome avec cette URL. Si l'URL est `https://hplanning2016.umons.ac.be/invite?fd=1`, pas de problème.